packages/data-schema/__tests__/__snapshots__/CustomOperations.test.ts.snap

```diff
@@ -258,8 +256,7 @@ type Subscription {
 `;
 
 exports[`CustomOperation transform dynamo schema custom subscriptions Custom subscription where .for() resource has a CustomType return type 1`] = `
-"type CreateCustomTypePostReturnType @aws_api_key
-{
+"type CreateCustomTypePostReturnType {
```

Comment:
Here and below, I'm seeing that the auth graphql directives (`@aws_api_key`,  `@aws_cognito_user_pools`, etc.) are now getting dropped from the generated types. 

Does this PR actually need to change anything about how return types are handled relative to `main`? I could be wrong, but AFAIK, we already support custom types and refs to custom types as the return types for custom queries and mutations.